index.rst

```diff
@@ -183,8 +209,33 @@ shall be ignored.
 Suggestions for Plugin Developers
 =================================
 
-TODO. For now please read the `Plugin Guidelines`_ on GitHub wiki. 
+TODO. For now please read the `Plugin Guidelines`_ on GitHub wiki.
+
+Versioning
+==========
+
+*This section applies beginning with v0.13 of this specification.*
+
+This specification has a version, tagged in the `specification repository`_.
+Each specification version corresponds to the same version in the
+`core-tests repository`_.
+
+Each EditorConfig core, to pass the core tests, must process version
+numbers given with the ``-b`` switch, and must report version numbers when
```

Comment:
I think this need more words to describe what is expected with this switch, wasn't clear to me until I read your comment on the PR. Also define what the core engine should do when it received a version number it can not use, what should it print, what's the required exit code?

Reply:
I agree.  Unfortunately, I'm not sure we have those answers yet :) .  Currently, there are only a few version-related checks in the core tests, and those are what this section is referring to.  If you are amenable, I will open a separate issue to discuss your point.  Please let me know your thoughts.

Reply:
+1 to continuing this in a new ticket and unblocking this PR. Look forward to the ticket!

Reply:
Open as https://github.com/editorconfig/editorconfig/issues/428